tasks: Update icon and presentation of tasks that were missing this. Also remove unused tasks

andreriesco opened the pull request.

Commit:
tasks: Update icon and presentation of tasks that were missing this

Also, remove duplicated pre-cleanup and pre-cleanup-debug.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby commented on assets/tasks/common.json:
```diff
@@ -731,78 +738,7 @@
-            }
-        },
-        {
-            "label": "pre-cleanup-debug",
```
these should not be removed I think ...

andreriesco replied:
It is actually still there down below, just removed because it was duplicated.

microhobby commented on assets/tasks/common.json:
```diff
@@ -285,38 +260,6 @@
             }
         },
-        {
-            "label": "docker-login",
```
The same here.

microhobby replied:
@andreriesco could you instead remove add a description and set the hide: false ?

andreriesco replied:
Sure

microhobby commented on assets/tasks/common.json:
```diff
@@ -132,31 +132,6 @@
         },
-        {
-            "label": "set-platform-fleet-name",
-            "detail": "",
```
this actually is a util task, it was set a hide true wrongly, maybe because was not description field.

andreriesco replied:
Sure, I will fix it

andreriesco replied:
This is being covered by the fill-pipeline-settings task now I believe

Commit:
tasks: Remove unused tasks

Criterial for removal: Tasks hidden (hide: true), which seem to not be
called anywhere in the extension's code or by any other task. Probably
old tasks that lost their usability with updates on other tasks or on
the extension's code.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>
